libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardLayout/utils/sizing.ts

```diff
@@ -376,13 +376,12 @@ export const getResizedItemPositions = <TWidget>(
         }, positions);
 };
 
-export const getDashboardLayoutItemHeight = (size: IDashboardLayoutSize, screen: ScreenSize) => {
-    const { heightAsRatio, gridHeight } = size;
+export const getDashboardLayoutHeight = (size: IDashboardLayoutSize): number | undefined => {
```

Comment:
Maybe we should name it `getDashboardLayoutItemHeight`. `getDashboardLayoutHeight` could be misleading, what do you think?

Reply:
Yes, makes more sense. Like it was before.